xbmc/addons/kodi-addon-dev-kit/include/kodi/xbmc_epg_types.h

```diff
@@ -81,7 +81,7 @@ extern "C" {
    */
   typedef struct EPG_TAG {
     unsigned int  iUniqueBroadcastId;  /*!< @brief (required) identifier for this event. Event uids must be unique for a channel. Valid uids must be greater than EPG_TAG_INVALID_UID. */
-    unsigned int  iUniqueChannelId;    /*!< @brief (required) unique identifier of the channel this event belongs to. */
+    int           iUniqueChannelId;    /*!< @brief (required) unique identifier of the channel this event belongs to. */
```

Comment:
Why are we changing this? What is -1 supposed to represent?

Reply:
It's PVR_CHANNEL_INVALID_UID for years. ;-)

=> https://github.com/xbmc/xbmc/blob/master/xbmc/addons/kodi-addon-dev-kit/include/kodi/xbmc_pvr_types.h#L182

Reply:
But isn't -1 for unsigned int INT_MAX? If that value is stored e.g. in the database it will overflow if we put it into an int.

Reply:
Not sure I understand what you mean. But I'm sure you have a valid point here and will drop this commit.

Reply:
I'm not sure I do, but yeah maybe best to drop it